docs/src/main/sphinx/release/release-478.md

```diff
@@ -0,0 +1,100 @@
+# Release 478 (dd Oct 2025)
+
+## General
+
+* {{breaking}} Move startup logs to `io.trino.bootstrap` package. ({issue}`26721`)
+* Run Trino on JDK 25.0.0 (build 36). ({issue}`26693`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Docker image
+
+## CLI
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+* Fix failure when reading `NULL` map on `json` type. ({issue}`26700`)
+
+## Druid connector
+
+## DuckDB connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Add support for reading encrypted Parquet files. ({issue}`24517`, {issue}`9383`)
+
+## Hudi connector
+
+## Iceberg connector
+
+* Fix failure due to column count mismatch when executing `add_files_from_table`
+  procedure. ({issue}`26774`)
+* Support using a local staging path for improved performance of writes to sorted
```

Comment:
Improve performance when writing sorted tables and the xxx config option is set.